easybuild/easyconfigs/a/Autocycler/Autocycler-0.5.0-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Cargo'
+
+name = 'Autocycler'
+version = '0.5.0'
+
+homepage = 'https://github.com/rrwick/Autocycler'
+description = """Autocycler is a tool for generating consensus long-read
+assemblies for bacterial genomes.  It is the successor to Trycycler, and for
+most users, I recommend Autocycler over Trycycler."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/rrwick/Autocycler/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['70efff324fc9b0a4ad70bef7984c4eb78b69b989011aa17eea4b5d75b5b9c5ff']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Rust', '1.83.0'),
+]
+
+offline = False
```

Comment:
This implies that this easyconfig will stop working in a while (couple of months, probably).

Any reason why you're using this, rather than includes a list of `crates`?
See also https://docs.easybuild.io/writing-easyconfig-files/#generate_crates_list

Reply:
Hi @boegel Do you mean `offline = False` is not really recommended? I just tried the step you suggested for generating a list of crates. The list is quite long (more than 200 items). I guess not all listed crates are needed. I will try removing `offline = False` and adding necessary crates later.

Reply:
Hi @Thyre Thank you for the comments. I was not familiar with the crates setting and now I get it. Now cartes added and also update the version to v0.5.2.